learning/tour-of-beam/frontend/assets/translations/en.yaml

```diff
@@ -0,0 +1,41 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+ui:
+  copyright: '© The Apache Software Foundation'
+  darkMode: 'Dark Mode'
+  lightMode: 'Light Mode'
+  privacyPolicy: 'Privacy Policy'
+  reportIssue: 'Report issue in Jira'
+  signIn: 'Sign in'
+  continueGitHub: 'Continue with GitHub'
+  continueGoogle: 'Continue with Google'
+pages:
+  home:
```

Comment:
Welcome was less ambiguous. Let's get it back. `splash` can also do.
"Home" is a role, and this can change. One day we may make the learning screen a home one.
Screens are better named after the content.

Reply:
I've renamed it in #226. I guess I should do a rebase.